builder/comp-builder.nix

```diff
@@ -235,7 +235,8 @@ stdenv.mkDerivation ({
 
   buildInputs = component.libs
     ++ component.frameworks
-    ++ builtins.concatLists component.pkgconfig;
+    ++ builtins.concatLists component.pkgconfig
+    ++ lib.optional (lib.strings.hasPrefix "gi-" fullName) gobject-introspection;
```

Comment:
This should have a comment.

```
# Note: This is a hack until we can properly solve pkgconfig propagated dependencies
```
or something. I'm going out on a limb here and assuming it's pkgconfig propagated dependencies?

Reply:
Comment added